webapp/fmprod/src/test/java/org/apache/oodt/cas/product/jaxrs/resources/MetadataResourceTest.java

```diff
@@ -65,7 +66,7 @@ public void testXmlMarshalling() throws IOException, JAXBException,
       + "</metadata>";
 
     // Create a MetadataResource using a Metadata instance.
-    Hashtable metadataEntries = new Hashtable<String, Object>();
+    HashMap<String, Object> metadataEntries = new HashMap<>();
```

Comment:
Same. If this change was intentional, we should be able to remove `HashTable` import